db/migrate/20150403192408_remove_deleted_aspects_from_auto_follow_back.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveDeletedAspectsFromAutoFollowBack < ActiveRecord::Migration
+  def up
+    User.where.not(auto_follow_back_aspect_id: Aspect.select(:id))
+        .where(auto_follow_back: true)
+        .update_all(auto_follow_back: false, auto_follow_back_aspect_id: nil)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.